<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemTool_BorderPanel</class>
 <widget class="QWidget" name="ItemTool_BorderPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="DB_CustomFrame_5" name="frame_border">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="DB_CustomPushButton_12" name="pBorderSize1">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DB_CustomPushButton_12" name="pBorderSize2">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DB_CustomPushButton_12" name="pBorderSize3">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DB_CustomPushButton_12" name="pBorderSize4">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_line">
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(224, 224, 224);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="DB_CustomPushButton_13" name="pBorderColor1">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="DB_CustomPushButton_13" name="pBorderColor3">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="DB_CustomPushButton_13" name="pBorderColor4">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="DB_CustomPushButton_13" name="pBorderColor2">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="DB_CustomPushButton_13" name="pBorderColor5">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="DB_CustomPushButton_13" name="pBorderColor6">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="DB_CustomPushButton_13" name="pBorderColor7">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="DB_CustomPushButton_13" name="pBorderColor8">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_CustomFrame_5</class>
   <extends>QFrame</extends>
   <header>CustomWidget/db_customframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomPushButton_12</class>
   <extends>QPushButton</extends>
   <header>CustomWidget/db_custompushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomPushButton_13</class>
   <extends>QPushButton</extends>
   <header>CustomWidget/db_custompushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
